lua/lib/renderer.lua

```diff
@@ -48,6 +48,21 @@ if icon_state.show_file_icon then
 
 end
 
+local get_symlink_icon = function() return icon_state.icons.symlink end
```

Comment:
this should return an empty string.

Reply:
This is the same logic as https://github.com/kyazdani42/nvim-tree.lua/blob/505d63a3e7253a78fcc3c94d9da415fde89e9887/lua/lib/renderer.lua#L32-L32, otherwise the default icon won't take effect when not using icons

Reply:
mmhh indeed, but that should not be expected behaviour. If a user doesn't want icons, we shouldn't display icons at all right ?

Reply:
They shouldn't get any icons from web-icons, and the defaults for icon.default and icon.symlink are empty. My main reason for defaulting to this value is to fix the padding when not using icons, this can be another setting (padding), but this approach is simple enough and it works